CMakeLists.txt

```diff
@@ -86,8 +86,13 @@ if(PROCPS_FOUND)
 	check_include_file(proc/devname.h HAVE_PROC_DEVNAME_H)
 endif()
 
-find_package(PythonLibs 2 EXACT)
-find_package(PythonInterp)
+# According to CMake documentation:
+# https://cmake.org/cmake/help/latest/module/FindPythonInterp.html
+# If calling both find_package(PythonInterp) and find_package(PythonLibs),
+# call find_package(PythonInterp) first to get the currently active Python
+# version by default with a consistent version of PYTHON_LIBRARIES.
+find_package(PythonInterp 3)
+find_package(PythonLibs 3 EXACT)
```

Comment:
what is the reason for EXACT?

Reply:
Difficult to tell, but the check finds any Python 3.y.z, so it is probably OK and doesn't make a difference. In theory, `find_package(PythonLibs 3)` should try to find Python libraries that are compatible with Python 3, and as Python 2 neither Python 4 are compatible, the behavior of the check is somehow expected.

Reply:
well yeah but if EXACT is not needed let's not even have it there